manifests/base/server/deployment.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: burrito-server
+  labels:
+    app.kubernetes.io/component: server
+    app.kubernetes.io/name: burrito-server
+    app.kubernetes.io/part-of: burrito
+  namespace: burrito
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: burrito-server
+  replicas: 1
+  template:
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/default-container: burrito
+      labels:
+        app.kubernetes.io/name: burrito-server
+    spec:
+      securityContext:
+        runAsNonRoot: true
+      containers:
+        - command:
+            - /burrito
+          args:
+            - server
+            - start
+          ports:
+            - containerPort: 8080
+              name: http
+          env:
+            - name: BURRITO_SERVER_WEBHOOK_GITHUB_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: burrito-webhook-secret
```

Comment:
shouldn't we provide this secret by default?

Reply:
How should we do it, can Kustomize generate random secrets? Or should we leave a placeholder value?

Reply:
ok fair point. LGTM :)

Reply:
Actually, I have already created this secret, I'm gonna put it in `envFrom`, I agree with you it's better than `env`